.github/workflows/ci.yaml

```diff
@@ -42,10 +42,9 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
       - name: Lint golang code
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v2.5.2
         with:
-          version: ${{ env.GOLANGCI_VERSION }}
-          args: -v
```

Comment:
How about keep the `-v` flag. It will be passed to golangci-lint cmd and make it output more info about errors if any. 
https://github.com/golangci/golangci-lint-action/blob/v3.1.0/action.yml#L8-L11